.travis.yml

```diff
@@ -5,12 +5,18 @@ sudo: false
 rvm:
 - "2.2.2"
 - "2.3.1"
+- "2.4.0"
 
 gemfile:
 - gemfiles/activesupport42.gemfile
 - gemfiles/activesupport50.gemfile
 - gemfiles/activesupport_master.gemfile
 
+matrix:
+  exclude:
+    - rvm: "2.4.0"
+      gemfile: gemfiles/activesupport42.gemfile
```

Comment:
Cannot install JSON which is used by 4.2 at this version. Skip this permutation.